transformation/src/main/java/org/opensearch/migrations/VersionMatchers.java

```diff
@@ -12,6 +12,12 @@ public class VersionMatchers {
     public static final Predicate<Version> isES_7_X = VersionMatchers.matchesMajorVersion(Version.fromString("ES 7.10"));
     public static final Predicate<Version> isES_7_10 = VersionMatchers.matchesMinorVersion(Version.fromString("ES 7.10.2"));
     public static final Predicate<Version> isES_8_X = VersionMatchers.matchesMajorVersion(Version.fromString("ES 8.17"));
+    public static final Predicate<Version> isES_7_0_to_7_9 =
+            v -> v != null
```

Comment:
Lets use or create a helper function instead of doing this inline like is done with the other matchers